beacon-chain/blockchain/process_block.go

```diff
@@ -501,6 +501,15 @@ func (s *Service) handleEpochBoundary(ctx context.Context, postState state.Beaco
 		if err := helpers.UpdateProposerIndicesInCache(ctx, copied); err != nil {
 			return err
 		}
+
+		if err := copied.SetSlot(copied.Slot() + params.BeaconConfig().SlotsPerEpoch); err != nil {
+			return err
+		}
+		// This is a no-op for caching proposer indices for epoch + 1.
+		_, err = helpers.BeaconProposerIndex(ctx, copied)
```

Comment:
Can't we simply call `UpdateProposerIndicesInCache(ctx, copied)` instead?

Reply:
cca618e505f3386013955aa0826d2b357f799963